client/extensions/woocommerce/woocommerce-services/state/shipping-label/actions.js

```diff
@@ -179,6 +194,31 @@ const convertToApiPackage = pckg => {
 		'weight',
 		'signature',
 	] );
+	if ( customsItems ) {
+		apiPckg.contents_type =
```

Comment:
There should be some unit tests for `convertToApiPackage`. It's a prime candidate for testing - a pure function that can get pretty complicated. That will help to make sure that everything works as expected in this change, in future changes, and it helps to document what behaviour we expect.

I'm going to list some issues I noticed in context below that could be useful for tests.